### ISSUE TITLE
[Bug]: ads bypass blocker

### ISSUE THREAD

**Issue by AuracleTech** [th-ch/youtube-music#2020]
### Preflight Checklist

- [X] I use the latest version of YouTube Music (Application).
- [X] I have searched the [issue tracker](https://github.com/th-ch/youtube-music/issues) for a bug report that matches the one I want to file, without success.
- [X] I understand that **th-ch/youtube-music has NO affiliation with Google or YouTube**

### YouTube Music (Application) Version

3.3.6

### Checklists

- [ ] I use the portable version of the YouTube Music Application.
- [ ] I can reproduce this issue in the [official version of (WEB) YTM](https://music.youtube.com).

### What operating system are you using?

Windows

### Operating System Version

Windows 11 Pro 10.0.22631 Build 22631

### What CPU architecture are you using?

x64

### Last Known Working YouTube Music (Application) version

3.3.6

### Reproduction steps

1. Ad blocker

### Expected Behavior

Blocked ads

### Actual Behavior

Ads
Note : oddly enough ads are still blocked on brave ( browser )

### Enabled plugins

Default plugins
Precise volume

### Additional Information

![ads womp womp](https://github.com/th-ch/youtube-music/assets/40531871/85de1c79-16db-4a37-a26c-666a682968df)

**Comment by lihuelworks** [comment]
See here: https://github.com/th-ch/youtube-music/issues/1817#issuecomment-2049502523 for a solution. Reason being https://github.com/th-ch/youtube-music/issues/1817#issuecomment-2023438777

**Comment by AuracleTech** [comment]
> See here: [#1817 (comment)](https://github.com/th-ch/youtube-music/issues/1817#issuecomment-2049502523)

Ill do it manually until its fixed thanks

**Comment by AuracleTech** [comment]
![image](https://github.com/th-ch/youtube-music/assets/40531871/47c737d6-8c93-4a13-9bd4-f69dfd1a36d7)
Yeah no this maybe partially fixed the issue but yeah no

**Comment by rad10** [comment]
I can confirm that even with the blocklist, it still plays ads. Not sure why though

**Comment by swoh816** [comment]
@AuracleTech  @rad10 If you haven't, you should also switch on:
Plugins --> Ad Blocker --> Blocker --> With blocklists
in addition to changing the line as in https://github.com/th-ch/youtube-music/issues/1817#issuecomment-2049502523.

**Comment by AuracleTech** [comment]
> @AuracleTech @rad10 If you haven't, you should also switch on: Plugins --> Ad Blocker --> Blocker --> With blocklists in addition to changing the line as in [#1817 (comment)](https://github.com/th-ch/youtube-music/issues/1817#issuecomment-2049502523).

We did

**Comment by rad10** [comment]
> > @AuracleTech @rad10 If you haven't, you should also switch on: Plugins --> Ad Blocker --> Blocker --> With blocklists in addition to changing the line as in [#1817 (comment)](https://github.com/th-ch/youtube-music/issues/1817#issuecomment-2049502523).
> 
> We did as the conversation on top, blocklist does not block everything

Yup I followed that verbatim and added in the file as well as the file from the other issue

**Comment by JellyBrick** [comment]
Duplicate of #1817

**Comment by rad10** [comment]
> Duplicate of #1817

Is this really a duplicate? #1817 refers to the "inplayer" feature, but we showed that it affects blocklist as well.
